### PR TITLE
Change the peer-dependency React

### PR DESCRIPTION
**Pull request by arturparkhisenko**
Changelog:
- Changed the peer-dependency for the `react`, checked the support for it here https://semver.npmjs.com/

Closes the #17 

And as a part of the Hacktoberfest month, I would appreciate and like to ask you to add all the necessary labels(tags), please.
It is `hacktoberfest` topic to the repository and `hacktoberfest-accepted` if PR was accepted.

### CODE REVIEW

**Review by SeanMcP — COMMENTED** [review]
Looks good!

**Review by SeanMcP — CHANGES_REQUESTED** [review]
Sorry, I'm going to take back this acceptance for small request. Can you add a line to the `CHANGELOG.md` under updated for this? That way people can point to the release when this changed.

**Comment by arturparkhisenko** [comment]
@SeanMcP Changed it. And it was required to force-push after rebase because the branch was created before the previous PR with the changelog was merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Added the `CHANGELOG.md`
+- Changed peer-dependencies for React to include versions higher than 16
```
Comment (SeanMcP) [inline CHANGELOG.md]:
```suggestion
- Changed peer-dependencies for React to include versions greater than or equal to 16
```

**Review by SeanMcP — CHANGES_REQUESTED** [review]
Updated the wording a little bit. @arturparkhisenko does that look good?

**Review by SeanMcP — APPROVED** [review]
Looks good!

**Pull request merged**